### DISCUSSION TITLE
Does Neper run in parallel (multiple processors)?

### DISCUSSION THREAD

**Discussion by J-AKAZ** [neperfepx/neper#140] [Q&A] [answered]
Hello,
I would like to know if there is a way to perform mesh calculation using different processors (parallel calculation) in Neper?
Thank you

**Comment by rquey** [comment, marked as answer]
Neper is multithreaded (and uses all threads by default) but not parallelized.  This is what you see in Neper's header:
```
========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
[...]
Info   : Running on XX threads.
[...]
```
